sql/core/src/main/scala/org/apache/spark/sql/execution/CollectMetricsExec.scala

```diff
@@ -69,7 +69,7 @@ case class CollectMetricsExec(
       // - Performance issues due to excessive serialization.
       val updater = collector.copyAndReset()
       TaskContext.get().addTaskCompletionListener[Unit] { _ =>
-        collector.setState(updater)
```

Comment:
can we remove `setState` now?

Reply:
`setState` seems to be called only from `AggregatingAccumulatorSuite` aside from `CollectMetricsExec` so we can remove `setState` and replace it with `addState` in `AggregatingAccumulatorSuite`.